imports/api/lists/server/publications.js

```diff
@@ -2,7 +2,7 @@
 
 import { Meteor } from 'meteor/meteor';
 
-import { Lists } from '../lists.js';
+import { Lists } from '/imports/api/lists/lists.js';
```

Comment:
This is where this all-or-mostly absolute approach becomes less beneficial to me.  A single-level, relative traversal within the scope of the `lists` API directory is perfectly reasonable and much more portable, so I'd be in favor of not doing this (or anything like it).

You could easily move this `lists` API to another project with a different base structure without this change.

I get the intention of this, but I think it's important to highlight the strengths of multiple options and where they each have their strengths.